.github/dependabot.yml

```
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: '/'
+    schedule:
+      interval: weekly
```

Can be increased to Monthly, if you think weekly will be too noisy
```suggestion
      interval: monthly
```